app/workers/github_repos/repo_sync_worker.rb

```diff
@@ -26,7 +26,9 @@ def perform(repo_id)
           stargazers_count: fetched_repo.stargazers_count,
           info_hash: fetched_repo.to_hash,
         )
-        repo.user&.touch(:github_repos_updated_at)
+        if repo.user&.github_repos_updated_at&.before?(30.minutes.ago)
```

Comment:
Should we make 30.minutes a constant here?

Reply:
Updated!